tests/cypress/integration/cmsWorkflow/unlockResubmit.spec.ts

```diff
@@ -0,0 +1,67 @@
+import * as path from 'path'
+
+describe('dashboard', () => {
+    it('can unlock and resubmit', () => {
+        cy.logInAsStateUser()
+
+        // a submitted submission
+        cy.startNewContractAndRatesSubmission()
+
+        cy.fillOutBaseContractDetails()
+        cy.navigateForm('Continue')
+
+        cy.fillOutNewRateCertification()
+        cy.navigateForm('Continue')
+
+        cy.fillOutStateContact()
+        cy.fillOutActuaryContact()
+        cy.navigateForm('Continue')
+
+        cy.fillOutSupportingDocuments()
+        cy.navigateForm('Continue')
+        cy.findByRole('heading', {name: /Review and submit/}).should('exist')
+
+        // Store submission url for reference later
+        cy.location().then((fullUrl) => {
+
+            const reviewURL = fullUrl.toString()
+            fullUrl.pathname = path.dirname(fullUrl)
```

Comment:
cypress has a helper for this. would prefer we use that

Reply:
I’m not sure I follow. I’m using dirname to pop off the end of the URL (‘review-and-submit’). It is importing werid so I’d probably rather do something else.

Reply:
Missed this. `cy.location('pathname')` I thought would help but I see what you are doing now and I'm wrong.  if this was in app-web theres a bunch of url helpers in react-router but at this level i guess we do need a dependency